src/popover.ts

```diff
@@ -11,25 +11,25 @@ export class HoverEditor extends HoverPopover {
   onTarget: boolean;
   onHover: boolean;
   isPinned: boolean;
-  leaf: HoverLeaf;
   isDragging: boolean;
   isResizing: boolean;
   isMenuActive: boolean;
   parent: HoverParent;
   interact: Interactable;
-  plugin: HoverEditorPlugin;
   lockedOut: boolean;
   abortController: AbortController;
-  rootSplit: WorkspaceSplit;
+  rootSplit: WorkspaceSplit = new (
+    // the official API has no contructor for WorkspaceSplit
+    WorkspaceSplit as new(ws: Workspace, dir: string) => WorkspaceSplit
```

Comment:
Interesting. I've never seen this syntax before. Nice way to get around the lack of a constructor in the official typing.

Reply:
Unfortunately there's no way to merge a class constructor declaration in a .d.ts, so this was the only thing I could do that wasn't ts-ignore.